sftp-app-dependencies/pom.xml

```diff
@@ -18,7 +18,7 @@
 
 	<properties>
 		<sshd-core.version>1.6.0</sshd-core.version>
-		<sftp-source-version>2.0.3.BUILD-SNAPSHOT</sftp-source-version>
+		<sftp-source-version>2.1.0.BUILD-SNAPSHOT</sftp-source-version>
```

Comment:
I believe we can just use `${version}` instead of this extra property.
I mean there in all the dependencies from this project, e.g.:
```
	<dependency>
				<groupId>org.springframework.cloud.stream.app</groupId>
				<artifactId>sftp-app-starters-source-common</artifactId>
				<version>${version}</version>
			</dependency>
```